orders/src/main/java/com/prumoitstore/order/domain/Client.java

```diff
@@ -0,0 +1,37 @@
+package com.prumoitstore.order.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import javax.persistence.*;
+
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Builder
+@Table(name = "clients")
+public class Client {
+
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "Client_id")
+    private Integer id;
+    private String surname;
+    private String lastname;
+    private String email;
+    private String phone;
+    private String nif;
+
+    @OneToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "fk_address_id")
+    private Address address;
+
+    @OneToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "fk_order_detail_id")
+    private OrderDetail orderDetail;
```

Comment:
You don't need to add OrderDetails relation here, because you already added the client in the orderDetail class.